src/sgame/sg_bot_nav.cpp

```diff
@@ -569,8 +569,16 @@ static bool BotAvoidObstacles( gentity_t *self, glm::vec3 &dir )
 	return true;
 }
 
+// `dir` does not need to be normalized
 static void BotDirectionToUsercmd( gentity_t *self, const glm::vec3 &dir, usercmd_t *cmd )
 {
+	if ( glm::length2( glm::vec2(dir) ) < 1.0e-5f )
```

Comment:
Don't we have some epsilon defined already? Did you pick this number arbitrarily?

Reply:
It seems we have dozens of epsilons defined, though none particularly appropriate. In this case I think the epsilon could be many orders of magnitude bigger or smaller in either direction and not matter.